h2o-py/tests/testdir_jira/pyunit_pubdev_6599_h2o_automl_output.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import h2o
+from h2o.automl import H2OAutoML, get_automl
+from tests import pyunit_utils
+
+
+class H2OAutoMLOutputTest(unittest.TestCase):
```

Comment:
probably the first explicit usage of Py `unittest` in the entire project.
We should have started that long ago, thx!